contentcuration/contentcuration/frontend/shared/views/ResizableNavigationDrawer.vue

```diff
@@ -49,12 +52,24 @@
       temporary: {
         type: Boolean,
         default: false,
+        validate(temporary) {
+          return temporary !== this.permanent;
+        },
+      },
+      permanent: {
+        type: Boolean,
+        default: false,
+        validate(permanent) {
+          return permanent !== this.temporary;
+        },
       },
     },
     data() {
+      const localStorageName = this.localName + '-drawer-width';
       return {
         dragging: false,
-        width: 300,
+        width: parseFloat(localStorage[localStorageName]) || this.minWidth,
```

Comment:
Might be good to either set the minWidth to be higher or set it to the maxWidth by default. I realize that 10px seems like a really small starting point by default